x-pack/plugins/enterprise_search/public/applications/applications/components/engines/create_engine_flyout.tsx

```diff
@@ -129,10 +129,10 @@ export const CreateEngineFlyout = ({ onClose }: CreateEngineFlyoutProps) => {
       <EuiFlyoutBody>
         <EuiFlexGroup direction="column">
           <EuiFlexItem grow>
-            <EuiCallOut title="Technical Preview feature" color="warning" iconType="beaker">
+            <EuiCallOut title="Beta feature" color="warning" iconType="beaker">
```

Comment:
I _think_ this `title` should have had a translation 😬

Reply:
Good catch!

Reply:
Added in a3320f6df11786f6342c211c9338214eb9bffb5d